renpy/common/00layeredimage.rpy

```diff
@@ -954,12 +956,14 @@ python early in layeredimage:
             ll = l.subblock_lexer()
 
             while ll.advance():
-                if ll.keyword("attribute"):
-                    parse_attribute(ll, rv)
-                    continue
+                if not ll.keyword("pass"):
```

Comment:
This doesn't seem right. Something like: 

```python
if ll.keyword("pass"):
     ll.expect_eol()
     ll.expect_noblock("pass")
    continue
```

Seems much more correct. (And the same thing should be on line 997.)